Gemfile.jruby-3.1.lock.release

```diff
@@ -725,7 +725,7 @@ GEM
     multipart-post (2.4.1)
     murmurhash3 (0.1.6-java)
     mustache (0.99.8)
-    mustermann (3.0.0)
+    mustermann (3.0.1)
```

Comment:
This is likely a desirable change looking at https://bugs.ruby-lang.org/issues/19266, but I'd rather not include it so close to a Logstash GA

```suggestion
    mustermann (3.0.0)
```